resources/windows/wix/appdatafiles.wxs

```diff
@@ -0,0 +1,160 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <!--
+    Puppet Agent Specific Registry Entries
+    This overrides the generic file in the vanagon directory
+  -->
+  <Fragment>
+    <DirectoryRef Id="TARGETDIR">
+      <Directory Id="CommonAppDataFolder" Name="CommonAppData">
+        <Directory Id="APPDATADIR" Name="PuppetLabs">
+          <Directory Id="pxpAgentDataFolder" Name="pxp-agent">
+            <Directory Id="pxpAgentDataFolderVar" Name="var">
+              <Directory Id="pxpAgentDataFolderLog" Name="log" />
+              <Directory Id="pxpAgentDataFolderRun" Name="run" />
+              <Directory Id="pxpAgentDataFolderSpool" Name="spool" />
+            </Directory>
+            <Directory Id="pxpAgentDataFolderEtc" Name="etc">
+              <Directory Id="pxpAgentDataFolderModules" Name="modules"/>
+            </Directory>
+          </Directory>
+          <Directory Id="PuppetAppData" Name="puppet">
+            <Directory Id="PuppetConfDir" Name="etc"/>
+            <Directory Id="PuppetVarDir" Name="var"/>
+          </Directory>
+        </Directory>
```

Comment:
@McdonaldSeanp This line belongs just before line 45.
If you check the install you will see that  ```code```, ```facter``` and ```mcollective``` are being created under ```ProgramData``` and not ```PuppetLabs```